### PR TITLE
moderation, design & performance improvements

### PR DESCRIPTION
**Pull request by xyzeva**

### CODE REVIEW

File: src/routes/MainPage.svelte [hunk 1]
```diff
@@ -297,7 +300,7 @@
 					{/if}
 				</div>
 				{#if lyntOpened && selectedLynt}
-					<div class="mb-2 h-full w-full max-w-[530px] pb-10">
+					<div class="mb-2 h-full w-full 53max-w-[0px] pb-10">
```
Comment (GStudiosX2) [inline src/routes/MainPage.svelte:303]: What happened here ?

File: src/lib/moderation.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export const NSFW_ERROR = json(
+	{
+		error:
+			'It seems like you just uploaded an NSFW image. We do not allow this type of content, if you believe this is a mistake, please contact us.'
+	},
+	{ status: 400 }
+);
+
+tf.enableProdMode();
+const model = await nsfw.load();
```
Comment (GStudiosX2) [inline src/lib/moderation.ts:16]: You should make the model local `await nsfw.load(`file://.
/models/mobilenet_v2_mid/`, { type: 'graph' })`

You can get the files here:
https://github.com/infinitered/nsfwjs/tree/master/models/mobilenet_v2_mid

**Review by GStudiosX2 — CHANGES_REQUESTED** [review]

File: src/lib/moderation.ts (hunk 2 above)
Comment (xyzeva) [reply src/lib/moderation.ts:16]: why?

Comment (GStudiosX2) [reply src/lib/moderation.ts:16]: > why?

```The magic that powers NSFWJS is the NSFW detection model. By default, this node module is pulling from my S3, but I make no guarantees that I'll keep that download link available forever. It's best for the longevity of your project that you download and host your own version of the model files. You can then pass the relative URL to your hosted files in the load function. If you can come up with a way to bundle the model into the NPM package, I'd love to see a PR to this repo!```
That's what it says on the nsfwjs npm page

**Pull request merged**